server/start.go

```diff
@@ -229,11 +231,54 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 		return err
 	}
 
+	// Add the tx service to the gRPC router. We only need to register this
+	// service if gRPC is enabled
+	if config.GRPC.Enable {
+		// create tendermint client
+		// assumes the rpc listen address is where tendermint has its rpc server
+		rpcclient, err := rpchttp.New(ctx.Config.RPC.ListenAddress, "/websocket")
```

Comment:
Same for comment as here: https://github.com/cosmos/cosmos-sdk/pull/18110/files#r1358278029